Makefile

```diff
@@ -1,48 +1,36 @@
-current_user  := $(shell id -u)
-current_group := $(shell id -g)
-BUILD_DIR     := $(PWD)
-TMPDIR        := $(BUILD_DIR)/tmp
-COMPOSER_FLAGS :=
-DOCKER_FLAGS  := --interactive --tty
-TEST_DIR      :=
-REDUX_LOG     :=
-UNIQUE_APP_CONTAINER := $(shell uuidgen)-app
-MIGRATION_VERSION :=
-APP_ENV       := dev
-
-.DEFAULT_GOAL := ci
+current_user    := $(shell id -u)
+current_group   := $(shell id -g)
+BUILD_DIR       := $(PWD)
+DOCKER_FLAGS    := --interactive --tty
+DOCKER_IMAGE    := registry.gitlab.com/fun-tech/fundraising-frontend-docker
 
 install-php:
-	docker run --rm $(DOCKER_FLAGS) --volume $(BUILD_DIR):/app -w /app --volume ~/.composer:/composer --user $(current_user):$(current_group) composer install --ignore-platform-reqs $(COMPOSER_FLAGS)
+	docker run --rm $(DOCKER_FLAGS) --volume $(BUILD_DIR):/app -w /app --volume ~/.composer:/composer --user $(current_user):$(current_group) $(DOCKER_IMAGE):composer composer install $(COMPOSER_FLAGS)
 
 update-php:
-	docker run --rm $(DOCKER_FLAGS) --volume $(BUILD_DIR):/app -w /app --volume ~/.composer:/composer --user $(current_user):$(current_group) composer update --ignore-platform-reqs $(COMPOSER_FLAGS)
+	docker run --rm $(DOCKER_FLAGS) --volume $(BUILD_DIR):/app -w /app --volume ~/.composer:/composer --user $(current_user):$(current_group) $(DOCKER_IMAGE):composer composer update $(COMPOSER_FLAGS)
 
-test: covers phpunit
+ci: phpunit cs stan
 
-covers:
-	docker-compose run --rm --no-deps --name $(UNIQUE_APP_CONTAINER)-$@ app ./vendor/bin/covers-validator
+ci-with-coverage: phpunit-with-coverage cs stan
+
+test: phpunit
 
 phpunit:
-	docker-compose run --rm --name $(UNIQUE_APP_CONTAINER)-$@ app ./vendor/bin/phpunit $(TEST_DIR)
+	docker-compose run --rm --no-deps app ./vendor/bin/phpunit
 
 phpunit-with-coverage:
-	docker-compose -f docker-compose.yml -f docker-compose.debug.yml run --rm --name $(UNIQUE_APP_CONTAINER)-$@ app_debug ./vendor/bin/phpunit --configuration=phpunit.xml.dist --coverage-clover coverage.clover
+	docker-compose run --rm --no-deps -e XDEBUG_MODE=coverage app ./vendor/bin/phpunit
 
 cs:
-	docker-compose run --rm --no-deps --name $(UNIQUE_APP_CONTAINER)-$@ app ./vendor/bin/phpcs
+	docker-compose run --rm --no-deps app ./vendor/bin/phpcs
 
 fix-cs:
-	docker-compose run --rm --no-deps --name $(UNIQUE_APP_CONTAINER)-$@ app ./vendor/bin/phpcbf
+	docker-compose run --rm --no-deps app ./vendor/bin/phpcbf
 
 stan:
-	docker-compose run --rm --no-deps --name $(UNIQUE_APP_CONTAINER)-$@ app php -d memory_limit=-1 vendor/bin/phpstan analyse --level=7 --no-progress src/ tests/
-
-phpmd:
-	docker-compose run --rm --no-deps --name $(UNIQUE_APP_CONTAINER)-$@ app ./vendor/bin/phpmd src/ text phpmd.xml
-
-ci: covers phpunit cs stan
+	docker-compose run --rm --no-deps app ./vendor/bin/phpstan analyse --level=7 --no-progress src/ tests/
```

Comment:
Please do `php -d memory_limit=1G vendor/bin/phpstan` instead of `./vendor/bin/phpstan`, PHPStan is prone to failing if it doesn't get enough memory. I don't know if we just copied the previous Makefile from FundraisingFrontend (looks like it) and don't need to increase the memory limit here, but it can't hurt to do it and help our future selves.